### ISSUE TITLE
HTML report support

### ISSUE THREAD

**Issue by Vinaykumar13** [ben-manes/gradle-versions-plugin#275]
Hi Team,

As you have given how to extend the closure, can you integrate in the code itself, so that we can get more flexibility on report format json, xml and html?

looking forward.

**Comment by ben-manes** [comment]
Can you rephrase your request? I don’t think that I understand what you’d like.

**Comment by Vinaykumar13** [comment]
> Can you rephrase your request? I don’t think that I understand what you’d like.

just like json and xml, Can this plugin provide build-in support for HTML?

**Comment by ben-manes** [comment]
Oh, as you said it can be done using a closure. If you want a built in version, PRs are encouraged.
